dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java

```diff
@@ -152,10 +152,8 @@ public void doSaveProperties(long version) {
             if (!lockfile.exists()) {
                 lockfile.createNewFile();
             }
-            RandomAccessFile raf = new RandomAccessFile(lockfile, "rw");
-            try {
-                FileChannel channel = raf.getChannel();
-                try {
+            try (RandomAccessFile raf = new RandomAccessFile(lockfile, "rw")) {

```

Comment:
As there are multiple nested try block, I would recommend to refactor this to individual method for better visibility and navigation.